crates/utils/src/lib.rs

```diff
@@ -13,7 +13,6 @@ cfg_if! {
 }
 
 pub mod error;
-pub use error::LemmyErrorType;
```

Comment:
This PR removes a pointless re-export of LemmyErrorType at a different location.